### PR TITLE
Deploy Dedicated VPX KB into service tasks

### PR DESCRIPTION
**Pull request by chrislittle**
new:  Deploy Dedicated VPX KB into service tasks

### CODE REVIEW

File: Service Tasks/deploy-a-dedicated-citrix-vpx-appliance.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### Deploy a Load Balancing VLAN
+
+* Deploy a [dedicated load balancing vlan](../Network/creating-and-deleting-vlans.md) in your account within the appropriate data center.  We recommend the VPX reside in a dedicated VLAN which allows for maximum Firewall security control and scalability of VIPs.
+* Costs for VLANs can be found in our [Pricing Catalog](http://www.centurylinkcloud.com/pricing) or your CenturyLink Cloud MSA.
+* Once this job completes we recommend you [apply a friendly name](../Network/add-a-user-friendly-name-to-vlans.md) to this VLAN.
+
+In the sample below in CA3 we have a network created and named NLB_10.100.97.0/24.
+
+<img src="../images/deploy-a-dedicated-citrix-VPX-appliance-01.png"/>
```
Comment (nathanyoung) [inline Service Tasks/deploy-a-dedicated-citrix-vpx-appliance.md]: why not use markdown syntax for images?

Comment (chrislittle) [inline Service Tasks/deploy-a-dedicated-citrix-vpx-appliance.md]: i'll remove the / at the end of the png now

Comment (nathanyoung) [inline Service Tasks/deploy-a-dedicated-citrix-vpx-appliance.md]: markdown syntax for images look like this:

`image alt text describing what the image is for screen readers`

**Comment by chrislittle** [comment]
fixed up code

File: Service Tasks/deploy-a-dedicated-citrix-vpx-appliance.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+### Deploy a Load Balancing VLAN
+
+* Deploy a [dedicated load balancing vlan](../Network/creating-and-deleting-vlans.md) in your account within the appropriate data center.  We recommend the VPX reside in a dedicated VLAN which allows for maximum Firewall security control and scalability of VIPs.
+* Costs for VLANs can be found in our [Pricing Catalog](http://www.centurylinkcloud.com/pricing) or your CenturyLink Cloud MSA.
+* Once this job completes we recommend you [apply a friendly name](../Network/add-a-user-friendly-name-to-vlans.md) to this VLAN.
+
+In the sample below in CA3 we have a network created and named NLB_10.100.97.0/24.
+
+![Dedicated Load Balancing VLAN Image](img src="../images/deploy-a-dedicated-citrix-VPX-appliance-01.png")
```
Comment (nathanyoung) [inline Service Tasks/deploy-a-dedicated-citrix-vpx-appliance.md]: You're mixing html syntax and markdown syntax. 
Markdown syntax for images looks like this: `Dedicated Load Balancing VLAN Image`

This is why the images are broken in the preview.
HTML equivalant looks like this: ``

**Comment by chrislittle** [comment]
thx Nathan. totally spaced this. hope its good now

**Pull request merged**